### PR TITLE
feat(support): submit translations

### PR DESCRIPTION
**Pull request by FredericEspiau**

### CODE REVIEW

File: packages/manager/modules/support/src/tickets/new-ticket/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -114,6 +114,7 @@
   "ovhManagerSupport_new_creation_tip_forum": "Notre forum :",
   "ovhManagerSupport_new_creation_tip_2": "Vous avez trouvé la réponse à votre demande entre temps ?",
   "ovhManagerSupport_new_creation_tip_3": "Dans ce cas, nous vous invitons à <a ui-sref=\"{{ state }}\">clôturer votre ticket</a> directement depuis votre espace client.",
+  "ovhManagerSupport_new_creation_optional": "{{label}} (facultatif)",
```
Comment (jleveugle) [inline packages/manager/modules/support/src/tickets/new-ticket/translations/Messages_fr_FR.json]: Why do we need to translate `{{label}}`?

Comment (FredericEspiau) [reply packages/manager/modules/support/src/tickets/new-ticket/translations/Messages_fr_FR.json]: We don't, it's a variable 🤔 

It used this to convey context about how `optional` is going to be used

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_eu#6161.0 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_us#6161.0 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Comment by ovh-cds** [comment]
CDS Report deploy_uxci_ca#6161.0 ✘  
* deploy_uxci
  * deploy_uxci ✘

**Comment by FredericEspiau** [comment]
@marie-j good point, it's the result of having code written between tech talks sessions :)

I'll do this before the one shot ticket is prodded but not in this PR :)

File: packages/manager/modules/support/src/tickets/new-ticket/translations/Messages_fr_FR.json (hunk 1 above)
Comment (antleblanc) [reply packages/manager/modules/support/src/tickets/new-ticket/translations/Messages_fr_FR.json]: @FredericEspiau Thanks for your answer!

**Pull request merged**